aiken-verifier/aiken_halo2/lib/constants_budget_check.ak

```diff
@@ -0,0 +1,48 @@
+use aiken/crypto/bls12_381/g1.{generator as generatorG1}
+use aiken/crypto/bls12_381/g2.{generator as generatorG2}
+use verifier_key.{
+  f0_commitment, f10_commitment, f11_commitment, f12_commitment, f13_commitment,
+  f14_commitment, f15_commitment, f16_commitment, f17_commitment, f18_commitment,
+  f19_commitment, f1_commitment, f2_commitment, f3_commitment, f4_commitment,
+  f5_commitment, f6_commitment, f7_commitment, f8_commitment, f9_commitment,
+  g2_const, p0_commitment, p1_commitment, p2_commitment, p3_commitment,
+  p4_commitment, p5_commitment, p6_commitment, p7_commitment, p8_commitment,
+}
+
+// this test is for checking how much budget is used when G1 constants are used in script, assertions are only to force use of the constant
+// this test depends on data generated with cargo run --example atms_with_lookups
+// you have to run that command before running this tes
+test use_constants() {
```

Comment:
I guess this fits better to the `verifier_key.ak`. Why in separate file?

Reply:
my idea was to keep templates for verifier logic without test code, and have separate files with tests, but yes, I can move it to verifier key template